views/index.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCKTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name = "viewport" content="width=device-width, intial-scale=1.0">
+  <title>Chitter</title>
+</head>
+<body>
+  <h1>Welcome to Chitter</h1>
+  <form action="/peeps" method="get">
+  <button type="submit">See All Peeps</button>
+</form>
+  <form action="/peeps/new" method="get">
+    <button type="submit">Add Peep</button>
+  </form>
+  <form action="/signup" method="get">
+  <button type="submit">Sign Up</button>
+</form>
+</body>
+</html>
```

Comment:
Again add newline at the end - rubocop will probably miss this one so watch out!